sdk/eventhub/azure-eventhubs/setup.py

```diff
@@ -8,78 +8,78 @@
 
 import re
 import os.path
+import sys
 from io import open
 from setuptools import find_packages, setup
 
+
 # Change the PACKAGE_NAME only to change folder and different name
-PACKAGE_NAME = "azure-eventhubs"
-PACKAGE_PPRINT_NAME = "MyService Management"
+PACKAGE_NAME = "azure-eventhub"
+PACKAGE_PPRINT_NAME = "Event Hubs"
 
 # a-b-c => a/b/c
 package_folder_path = PACKAGE_NAME.replace('-', '/')
 # a-b-c => a.b.c
 namespace_name = PACKAGE_NAME.replace('-', '.')
 
-# azure v0.x is not compatible with this package
-# azure v0.x used to have a __version__ attribute (newer versions don't)
-try:
-    import azure
-    try:
-        ver = azure.__version__
-        raise Exception(
-            'This package is incompatible with azure=={}. '.format(ver) +
-            'Uninstall it with "pip uninstall azure".'
-        )
-    except AttributeError:
-        pass
-except ImportError:
-    pass
-
 # Version extraction inspired from 'requests'
-with open(os.path.join(package_folder_path, 'version.py'), 'r') as fd:
-    version = re.search(r'^VERSION\s*=\s*[\'"]([^\'"]*)[\'"]',
+with open(os.path.join(package_folder_path, '__init__.py'), 'r') as fd:
+    version = re.search(r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]',
                         fd.read(), re.MULTILINE).group(1)
 
 if not version:
     raise RuntimeError('Cannot find version information')
 
-with open('README.rst', encoding='utf-8') as f:
+with open('README.rst') as f:
     readme = f.read()
-with open('HISTORY.rst', encoding='utf-8') as f:
+with open('HISTORY.rst') as f:
     history = f.read()
 
+exclude_packages = [
+        'tests',
+        "tests.asynctests",
+        'examples',
+        # Exclude packages that will be covered by PEP420 or nspkg
+        'azure',
+    ]
+
+if sys.version_info[0] < 3 or (sys.version_info[0] == 3 and sys.version_info[1] < 5):
```

Comment:
Why not simply check against the minimum version? E.g. 

```python
if sys.version_info < (3, 5, 3):
    exclude packages.extend ...
```

Reply:
Yes this is more concise. but (3, 5) or (3, 5, 3)?